zerver/lib/display_recipient.py

```diff
@@ -122,101 +122,39 @@ def bulk_fetch_user_display_recipients(
     Returns dict mapping recipient_id to corresponding display_recipient
     """
 
-    if len(recipient_tuples) == 0:
-        return {}
```

Comment:
Why is this part removed?

Reply:
Oops, I restored it.  The zero-guard isn't technically necessary, but it's certainly worth skipping over all the logic in the relatively common case of zero user-based recipients.